app/models/profile.rb

```diff
@@ -17,12 +17,6 @@ def self.refresh_attributes!
     return unless Database.table_available?("profiles")
 
     ProfileField.find_each do |field|
-      # Don't generate accessors for static fields stored on the table.
```

Comment:
The corresponding fields have been removed, so we can delete this comment and the guard clause.